### DISCUSSION TITLE
Is there any GitHub repo for code benaming?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [community/community#49379] [Repositories]
There's  a saying,

> Naming is hard when you're writing code 

So, is there a GitHub repo for naming?

**Comment by tuves** [comment]
Welcome to the GitHub Community, USERNAME, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
